src/EditorFeatures/CSharpTest/GenerateDefaultConstructors/GenerateDefaultConstructorsTests.cs

```diff
@@ -728,10 +729,6 @@ public Program()
     {
     }
 
-    public Program()
```

Comment:
This is good.  If you expand the code above htis, you'll see we were generating the default constructor twice.